spack_configs/blueos_3_ppc64le_ib/compilers.yaml

```diff
@@ -26,10 +26,10 @@ compilers:
     environment: {}
     extra_rpaths: []
 - compiler:
-    spec: clang@8.0.1
+    spec: clang@9.0.0
```

Comment:
RAJA is using chang@8.0.1 in its scripts, can you confirm they won’t need it?

Reply:
They will be using the same configs as Umpire 😄 but I think we should keep 8.0.1 too.

Reply:
You tell me. At this point we don’t have to keep more config than we need.

Reply:
I've added CUDA 10.1.243, but won't do clang 8. I want the RAJA jobs to match the Umpire ones.